Doc/whatsnew/3.14.rst

```diff
@@ -464,6 +464,9 @@ Other language changes
   of HMAC is not available.
   (Contributed by Bénédikt Tran in :gh:`99108`.)
 
+* When subclassing from a pure C type, the C slots for the new type are no
+  longer replaced with a wrapped version on class creation if they are not
+  explicitly overridden in the subclass.
```

Comment:
(Contributed by .. in ..)

(So there's a reference to the issue and from there the PR).